platform/features/layout/src/LayoutController.js

```diff
@@ -440,6 +498,37 @@ define(
             return this.gridSize;
         };
 
+        /**
+         * Gets the selection context.
+         *
+         * @param domainObject the domain object
+         * @returns {object} the context object which includes
+         *                  item, oldItem and toolbar
+         */
+        LayoutController.prototype.getContext = function (domainObject, toolbar) {
+            return {
```

Comment:
Since `getContext` is called from the template, this might be a performance concern; I believe this will return a new object (and call `useCapability` and `getToolbar`) every digest cycle. I recommend profiling a layout with a lot of frames to see if this shows up.